.golangci.yml

```diff
@@ -147,6 +147,7 @@ linters:
     - gochecknoglobals
     - gocognit
     - gocyclo
+    - godot
```

Comment:
N.B.: I've disabled [godot](https://golangci-lint.run/usage/linters/#godot), because:

- it doesn't seem to work all the time - sometimes it asks for full stop, other times it doesn't, without any apparent pattern
- some comments really don't require to be a full sentence imo

Reply:
```
const (
	// boolean tests
	testVerdictOk testVerdict = "OK"

	// measurement tests
	testVerdictGood testVerdict = "Good"
	testVerdictAvg  testVerdict = "Avg"
	testVerdictBad  testVerdict = "Bad"

	// failed tests
	testVerdictFail testVerdict = "Fail"
)
```

For this code snippet it forces full stop after `// failed tests`, but not after the others. I also see on some places in the code full stops arbitrary put on top of function names, on top of variables or on the same line (i.e.: `p2p/bootnode.go`).